sirepo/simulation_db.py

```diff
@@ -526,6 +523,33 @@ def read_simulation_json(sim_type, sid, qcall):
     return d
 
 
+def random_id(parent_dir, simulation_type=None):
```

Comment:
`mkdir_random` is a better name now that it is a public method.

Reply:
changed with 4aaf2e6